File: modul-06/PRAK604-2410817310014-INDRA_SURYADILAGA.py
sandi = str(input())
pesan = str(input())

if len(sandi) != len(pesan):
    print("Panjang kalimat berbeda, pesan palsu")
else:
    bintang = 0
    pagar = 0

    for i in range(len(sandi)):
        if sandi[i] == pesan[i]:
            if sandi[i] == ' ':
                print(" ", end="")
            else :
                print("*", end="")
                bintang += 1
        else :
            print("#", end="")
            pagar += 1

    print(f"\n* = {bintang}")
    print(f"# = {pagar}")

    if bintang >= pagar:
        print("Pesan asli")
    else:
        print("Pesan palsu")

File: modul-06/PRAK604-2410817310014-INDRA_SURYADILAGA.c
#include <stdio.h>
#include <string.h>

int main() {
    char sandi[100], pesan[100];
    fgets(sandi, sizeof(sandi), stdin);
    fgets(pesan, sizeof(pesan), stdin);

    if (strlen(sandi) != strlen(pesan)) {
        printf("Panjang kalimat berbeda, pesan palsu");
        return 0;
    }

    int bintang = 0, pagar = 0;
    for (int i=0; i < strlen(sandi)-1; i++) {
        if (sandi[i] == pesan[i]) {
            if (sandi[i] == ' ') {
                printf(" ");
            } else {
                printf("*");
                bintang += 1;
            }
        } else {
            printf("#");
            pagar += 1;
        }
    }
       
    printf("\n* = %d", bintang);
    printf("\n# = %d\n", pagar);
    if (bintang >= pagar) {
        printf("Pesan asli\n");
    } else {
        printf("Pesan palsu");
    }
    return 0;
}